lib/core-integration/src/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -1527,6 +1521,33 @@ spec = do
 
     transactionDeleteTest04 "byron-wallets"
 
+    transactionDeleteTest07 emptyWallet "wallets"
+
+    transactionDeleteTest07 emptyByronWallet "byron-wallets"
+
+    transactionDeleteTest08 emptyWallet "wallets"
+
+    transactionDeleteTest08 emptyByronWallet "byron-wallets"
+
+    transactionDeleteTest09 "wallets"
+
+    transactionDeleteTest09 "byron-wallets"
+
+    it "TRANS_DELETE_06 -\
```

Comment:
why can't it be also in "byron-wallets" ?

Reply:
and why we omitted test number 5 ;-) ?

Reply:
> why can't it be also in "byron-wallets" ?

I think that it cannot be mapped 1-1 like other tests because on byron you cannod create transaction but only migration, which basically selects all the funds and performs several txs. So the test would be slightly different. @jonathanknowles is working on a pr about integration tests for migration, so I'm thinking to leave it actually to another pr (perhaps after his pr is merged). 
BTW. I think that's be the same for `transactionDeleteTest01` and `transactionDeleteTest02`, which need to be reworked also.

>and why we omitted test number 5 ;-) ?

Well, I just mapped the requirements like that :) -> https://docs.google.com/spreadsheets/d/1fadIA_j4nCd3FbylPo5J8K9Fy6tixSC2T9V3xHKuUvU/edit
and turned out that `TRANS_DELETE_05` is in `lib/core-integration/src/Test/Integration/Scenario/API/Transactions.hs` (cause it fitted better there).